mdpow/tests/test_dihedral.py

```diff
@@ -23,9 +23,9 @@
 
 class TestDihedral(object):
     DG48910_mean = -172.9849512527183
-    DG491011_mean = -3.7300197060478695
-    DG48910_var = 1490.6576365537262
-    DG491011_var = 128.3805265432388
+    DG491011_mean = 177.74725233051953
+    DG48910_var = 0.20311120667628546
+    DG491011_var = 0.006976126708773456
```

Comment:
revert changes, will be fixed in PR #218

Reply:
new values may still required, circvar should always be between 0 and 1 per the scipy docs